### ISSUE TITLE
Add control api to coordinator

### ISSUE THREAD

**Issue by bonomat** [get10101/10101#604]
We should add an API to the coordinator which 

1. enables/disables JIT channels: this is crucial at times like these where the mempool is going nuts. At times like these we do not want to pay high fees. It's better to just wait till things normalize
2. enable/disables new positions: sometimes we need to roll-out breaking changes. We can't do this safely if we have positions open. Disabling new positions for 24h would allow us to ensure that we have no new positions at some point in the future at which we then can roll out a new version.

**Comment by klochowicz** [comment]
@bonomat how should we communicate to the users in-app that the coordinator is not accepting new orders? 
I'd imagine it would be very surprising if this didn't work for the user.

**Comment by bonomat** [comment]
I guess there are multiple ways you can deal with this. The most honest one would be to add a warning we are in maintainance mode and a new update will be available soon.

**Comment by holzeis** [comment]
> Disabling new positions for 24h would allow us to ensure that we have no new positions at some point in the future at which we then can roll out a new version.

We'd need to disable new positions for 48 hours and also force close any remaining open position after that. (As we can't guarantee that the user comes back online within that duration)

**Comment by holzeis** [comment]
> I guess there are multiple ways you can deal with this. The most honest one would be to add a warning we are in maintainance mode and a new update will be available soon.

I think short term, a tweet should be sufficient?

On a side note, we have to make sure we are only blocking opening new positions, closing positions should always be possible. That means we can't simply stop the maker and let the orders fail, we have to check on the coordinator, if the user is attempting to open or close a position.

**Comment by klochowicz** [comment]
> 

some of the users might not be active tweeter users. if you register by email, maybe an email update would be best.

**Comment by klochowicz** [comment]
> On a side note, we have to make sure we are only blocking opening new positions, closing positions should always be possible. That means we can't simply stop the maker and let the orders fail, we have to check on the coordinator, if the user is attempting to open or close a position.

Correct; that's why the fix requires code change, otherwise we could just shut down some services for that time.

**Comment by klochowicz** [comment]
@bonomat how much do we want all of these to be configurable instantly at runtime?
The following ones are pretty hard to pull off (probably possible, but not trivial):
- enable/disable positions
- tx fees 

I can really easily make them applied once during startup. The thing is that these parameters live really deep in our shared state, and in multiple threads.
I think I can pull this off with `RwLock` and shared settings, but it might take longer than expected. Thoughts?
